src/connectService.ts

```diff
@@ -204,6 +239,15 @@ class ConnectService extends IframeService {
     this.storeAccount(accountId, toHex(entropy))
     return entropy
   }
+  /**
+   *  Creates a legacy 3Box root seed
+   */
+  async legacySeedCreate(): Promise<Uint8Array> {
+    const message = 'This app wants to view and update your 3Box profile.'
+    const authSecret = await this.authenticate(message)
+    const seed = mnemonicToSeed(entropyToMnemonic(authSecret))
```

Comment:
This seems weird. I guess it could be ok since we already use the hdnode lib in 3id-did-provider.